WolvenKit.App/Controllers/RED4Controller.cs

```diff
@@ -430,16 +430,30 @@ public async Task<bool> LaunchProject(LaunchProfile options)
         _progressService.IsIndeterminate = false;
 
         // launch game
-        if (options.LaunchGame)
+        if (!options.LaunchGame)
+        {
+            return true;
+        }
+
+        if (_settingsManager.GetRED4GameExecutablePath() is not string launchCommand || string.IsNullOrEmpty(launchCommand))
```

Comment:
Using `CP77ExecutablePath` as a fallback for `GetRED4GameLaunchCommand()` looks ok. But why remove the usage then here?